app/api/store/route.js

```diff
@@ -0,0 +1,36 @@
+// This is an example for an API endpoint that serves sensitive information for an authenticated user's session
+import { AddDOB } from '../../../lib/db';
+import { verifyTideCloakToken } from '/lib/tideJWT';
+
+// This endpoint is validating that only a specific role will be authorised to access the data
+const AllowedRole = '_tide_dob.selfencrypt';
+
+// base URL from the local .env (http://localhost:3000)
+const origin = process.env.BASE_URL;
```

Comment:
```suggestion
const origin = process.env.BASE_URL ?? "http://localhost:3000";
```

called multiple times, move to constants file